ee/ucp/interlock/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Layer 7 routing overview
+description: Learn how to route layer 7 traffic to your Swarm services
+keywords: routing, UCP, interlock, load balancing
+---
+
+Application-layer (Layer 7) routing is the application routing and load balancing (ingress routing) system included with Docker Enterprise for Swarm orchestration. Interlock architecture takes advantage of the underlying Swarm components to provide scalable Layer 7 routing and Layer 4 VIP mode functionality.
```

Comment:
```suggestion
Application-layer (Layer 7) routing is the application routing and ingress load balancing system included with Docker Enterprise for Swarm orchestration. Interlock architecture takes advantage of the underlying Swarm to provide scalable Layer 7 routing for Swarm services.
```

- reduce parenthetical stutter in the first sentence
- remove reference to layer 4.  this is about layer 7, right?

Reply:
please dear deity introduce interlock before you start talking about its architecture.